### PR TITLE
Document container image packaging

### PR DESCRIPTION
**Pull request by mangelajo**

### CODE REVIEW

File: content/en/docs/user-documentation/how-tos/offline-containers.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+draft: false
+weight: 9
+tags:
+  - Container Storage
+  - Disconnected
+description: Offline containers are containers which are stored in the operating system image, and made available to `cri-o` via the `/etc/container/storage.conf` `additionalimagestores` list.
+---
+## What are offline container images
+
+Offline containers are containers which are stored in the operating system image,
```
Comment (copejon) [inline content/en/docs/user-documentation/how-tos/offline-containers.md:12]: "system image" sounds more specific to rpm-ostree installs than standard installs.

File: content/en/docs/developer-documentation/build-install-container.md [hunk 2]
```diff
@@ -66,3 +66,27 @@ Substitute the image name:tag in the podman command with the newly built image n
...
+```bash
+cd packaging/rpm
+./make-microshift-images-rpm.sh
+```
+
+or
+
+```bash
+cd packaging/rpm
+export COPR_REPO=mangelajo/microshift-containers
```
[the remark this answers is not in the document]
Comment (mangelajo) [reply content/en/docs/developer-documentation/build-install-container.md:90]: It's an example of override, the default is redhat-et, for now.

May be I should explain that.

File: content/en/docs/user-documentation/how-tos/offline-containers.md (hunk 1 above)
Comment (mangelajo) [reply content/en/docs/user-documentation/how-tos/offline-containers.md:12]: I was explicitly thinking of that, may be I should extend the explanation to make it clearer it works for both.

File: content/en/docs/developer-documentation/build-install-container.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (oglok) [reply content/en/docs/developer-documentation/build-install-container.md:90]: +1

File: content/en/docs/user-documentation/how-tos/offline-containers.md (hunk 1 above)
Comment (oglok) [reply content/en/docs/user-documentation/how-tos/offline-containers.md:12]: +1

File: content/en/docs/user-documentation/how-tos/offline-containers.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+and made available to `cri-o` via the `/etc/container/storage.conf` `additionalimagestores`
+list.
+
+Those container images are accesible for `cri-o` to create containers. Those images
+cannot be deleted, but newer versions of those containers can be downloaded normally,
+which `cri-o` will store in the general R/W container storage of the system.
+
+## When to use offline container images
+
+Offline containers are useful when the Edge device will have restricted connectivity,
```
Comment (oglok) [inline content/en/docs/user-documentation/how-tos/offline-containers.md:22]: s/Edge/edge

File: content/en/docs/user-documentation/how-tos/offline-containers.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+To install the MicroShift container images you can use:
+```bash
+curl -L -o /etc/yum.repos.d/microshift-containers.repo \
+          https://copr.fedorainfracloud.org/coprs/g/redhat-et/microshift-containers/repo/fedora-35/group_redhat-et-microshift-containers-fedora-35.repo
+
+rpm-ostree install microshift-containers
+````
+
+Or simply install that repository and package from an os build blueprint.
```
Comment (oglok) [inline content/en/docs/user-documentation/how-tos/offline-containers.md:48]: Do you mean to install the `microshift-containers` RPM in an rpm-ostree base OS image using something like Image Builder?  or do you mean OSbuild as a tool? 

I'd make this one sentence a bit more clear.

File: content/en/docs/user-documentation/how-tos/offline-containers.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+The tool can produce an srpm, rpm, or push a build to a copr repository.
+
+Some example usages:
+
+```bash
+./paack.py rpm example-user-containers.yaml centos-stream-9-aarch64
+```
+
+The target os is not important (`centos-stream-9`) but we need one os target
```
Comment (oglok) [inline content/en/docs/user-documentation/how-tos/offline-containers.md:65]: s/os/OS

File: content/en/docs/user-documentation/how-tos/offline-containers.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+compatible with the destination architecture.
+
+```bash
+./paack.py srpm example-user-containers.yaml
+```
+
+The produced `srpm` format contains the repository binaries and manifests for each architecture,
+then the build system unpacks the specific architecture for the build. The post install step
+of rpm configures the additionalimagestores in `/etc/container/storage.conf`
```
Comment (oglok) [inline content/en/docs/user-documentation/how-tos/offline-containers.md:74]: s/additionalimagestores/`additionalimagestores`

**Review by oglok — CHANGES_REQUESTED** [review]

**Pull request merged**